src/StackExchange.Redis/Enums/HashFieldFlags.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace StackExchange.Redis;
+
+/// <summary>
+///  Specifies the options to the HSETF command when to create hash/fields and the return data
+/// </summary>
+[Flags]
+public enum HashFieldFlags
+{
+
+    /// <summary>
+    /// No options specified.
+    /// </summary>
+    None = 0,
+    /// <summary>
+    /// When DC (“Don’t Create”) is specified: if key does not exist: do nothing (don’t create key)  
+    /// </summary>
+    DC = 1,
+    /// <summary>
+    ///  When DCF (“Don’t Create Fields”) is specified: for each specified field: if the field already exists: set the field's value and expiration time; ignore fields that do not exist
+    /// </summary>
+    DCF = 2,
+    /// <summary>
+    ///   When DOF (“Don’t Overwrite Fields”) is specified: for each specified field: if such field does not exist: create field and set its value and expiration time; ignore fields that already exists  
+    /// </summary>
+    DOF = 4,
+    /// <summary>
+    /// When GETNEW is specified: returns the new value of given fields  
+    /// </summary>
+    GETNEW = 8,
+    /// <summary>
+    /// When GETOLD is specified: returns the old value of given fields  
+    /// </summary>
+    GETOLD = 16,
+}
+
+internal static class HashFieldFlagsExtensions
+{
+    internal static bool isNone(this HashFieldFlags flags) =>
```

Comment:
.NET naming conventions are Proper case; I also wonder whether a single `.HasAny(this value, flag) => (value & flag) != 0;` achieves more with less

Reply:
fixed